sym_api_client_python/datafeed_event_service.py

```diff
@@ -270,6 +272,10 @@ async def handle_datafeed_errors(self, thrown_exception):
         """
         try:
             raise thrown_exception
+        except CancelledError as exc:
+            log.info("Cancel request received. Stopping datafeed...")
+            await self.deactivate_datafeed()
```

Comment:
Do we want for the pending query to complete here? Or could it be faster to pass wait_for_handler_completions=False?